devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -942,7 +961,7 @@ bool WholeBodyDynamicsDevice::applyLPFSettingsFromConfig(const yarp::os::Propert
     {        
         return false;
     }
-    
+       
```

Comment:
```suggestion

```

Reply:
Addressed in https://github.com/robotology/whole-body-estimators/pull/72/commits/b495053958d918d01711010f7e32f1ac651fc29c